cli/cmd/run.go

```diff
@@ -0,0 +1,15 @@
+package cmd
```

Comment:
Is this a work in progress?

Reply:
No. Actually I will update it in next push.